src/components/FullWidthPromo/FullWidthPromo.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import { ImageElement } from 'Image';
+import Button from 'Button';
+import Icon from 'Icon';
+import Link from 'Link';
+import RichText from 'RichText';
+import TagList, { TagProps } from 'TagList/TagList';
+
+type FullWidthPromoProps = {
+  authors?: string[];
+  description?: string;
+  href: string;
+  imageAlt?: string;
+  imageCaption?: string;
+  imageCredit?: string;
+  imageLicence?: string;
+  imageSizes?: string;
+  imageSrc?: string;
+  imageSrcSet?: string;
+  linkText: string;
+  metaLabel?: string;
+  title: string;
+  topics: TagProps[];
+};
+
+export const FullWidthPromo = ({
+  authors,
+  description,
+  href,
+  imageAlt,
+  imageSizes,
+  imageSrc,
+  imageSrcSet,
+  linkText,
+  metaLabel,
+  title,
+  topics
+}: FullWidthPromoProps) => (
+  <div className="cc-fw-promo">
+    {(!!imageSrc || (imageSrcSet && imageSizes)) && (
+      <figure className="cc-fw-promo__image">
+        <ImageElement src={imageSrc} srcSet={imageSrcSet} alt={imageAlt} />
```

Comment:
Do you think we should provide `height` and `width` to the ImageElement?

Should help if/when we migrate to the NextJS Image component. Regardless, it would also help with lazy-loading if/when we implement that, too.

Reply:
Done